site/docs/guides/create-derivation.md

```diff
@@ -0,0 +1,251 @@
+If you need help, see the [guide to create a Data Flow](./create-dataflow.md).
+
+* **flowctl** installed locally. For help, see the [installation instructions](../concepts/flowctl.md#installation-and-setup).
+
+## Pull your specification files locally
+
+To begin working in your local environment, you must authenticate Flow from the command line.
+Then, you'll need to add your source collection's specification files to a **draft** and copy the draft locally.
+
+1. Go to the [Flow web application](https://dashboard.estuary.dev). On the **Admin** page, click the **CLI-API** tab and copy the access token.
+
+2. In your local environment, authenticate with the token:
+
+   ```console
+   flowctl auth token --token ${your-token-here}
+   ```
+
+   The output message `Configured access token` indicates that you can now work with your Flow account in your local environment,
+   using the capabilities you've been provisioned.
+   To learn more about capabilities and permissions, see the [authorization documentation](../reference/authentication.md).
+
+3. Begin by creating a fresh draft. This is where you'll add the specification files you need from the catalog.
+
+   ```console
+   flowctl draft create
+   ```
+
+   The output table shows the draft ID and creation time. It doesn't have any catalog entities in it yet.
+   You'll add the source collection's specification to the draft.
+
+4. Identify the collection (or collections) in the catalog that contains the data you want to derive, and add it to your draft.
+
+   ```console
+   flowctl catalog draft --name namespace/data-flow-name/my-collection
+   ```
+
+   :::tip
+   If you're unsure of the name the collection, check **Collections** page in the web application.
+   You can also use `flowctl draft list` locally
+   to begin exploring the catalog items available to you.
+
+   The name of your collection may not follow the structure of the examples provided;
+   simply copy the entire name as you see it, including all prefixes.
+   :::
+
+   The output confirms that the entity name you specified has been added to your draft, and is of the type `collection`.
+
+   Your draft is set up, but still exists only on the Estuary servers.
+
+5. Pull the draft locally to edit the specification files.
+
+   ```console
+   flowctl draft develop
+   ```
+
+   The specification files are written to your working directory.
+
+## Add a derivation
+
+1. Locate the specification YAML file for the collection you want to transform.
+
+   In your working directory, you'll see a top-level file called `flow.yaml`.
+   Within a subdirectory that shares the name of your Data Flow, you'll find a second `flow.yaml` — this contains the collection specification.
+
+   :::tip
+   Use `tree` to visualize your current working directory. This is a helpful tool to understand the [files that underlie your local draft](../concepts/flowctl.md#development-directories).
```

Comment:
I might include an example output of `tree` in a draft directory here, just so people have something to visualize the directory structure. I know that'd be useful for me at least